<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Дисциплины</string>
       </attribute>
       <widget class="QTableView" name="Disc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>341</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Add_Disc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_Disc">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_Disc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_Disc">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дисциплины</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Циклы дисциплин</string>
       </attribute>
       <widget class="QTableView" name="cDisc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>341</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_cDisc">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Циклы дисциплин</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_cDisc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_cDisc">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Add_cDisc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Специальности</string>
       </attribute>
       <widget class="QTableView" name="Spec">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>341</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_Spec">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Специальности</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_Spec">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_Spec">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Add_Spec">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Кафедры</string>
       </attribute>
       <widget class="QPushButton" name="Add_Kaf">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_Kaf">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QTableView" name="Kaf">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>341</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_Kaf">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>280</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Кафедры</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_Kaf">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Дисциплины по циклам</string>
       </attribute>
       <widget class="QTableView" name="Disc_CDisc">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>391</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="Disc_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>311</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дисциплины по циклам</string>
        </property>
       </widget>
       <widget class="QTableView" name="cDisc_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>200</y>
          <width>311</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Add_Disc_CDisc">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_Disc_CDisc">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>380</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_Disc_CDisc">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_Disc_CDisc">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Дисциплины по кафедрам</string>
       </attribute>
       <widget class="QTableView" name="All_Kod">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>401</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дисциплины по кафедрам</string>
        </property>
       </widget>
       <widget class="QTableView" name="Kaf_2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>20</y>
          <width>441</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="cDisc_3">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>220</y>
          <width>251</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="Disc_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>220</y>
          <width>231</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Add_All_Kod">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OK_All_Kod">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>400</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Cancle_All_Kod">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Del_All_Kod">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Отчёт</string>
       </attribute>
       <widget class="QPushButton" name="pbGenerateReport">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Генерация отчёта</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
